kerykeion/schemas/kr_models.py

```
@@ -242,6 +250,12 @@ class MoonPhaseUpcomingPhasesModel(SubscriptableBaseModel):
 class MoonPhaseIlluminationDetailsModel(SubscriptableBaseModel):
     """
     Numeric illumination details for the Moon at the given moment.
+
+    Note:
+        ``phase_angle`` holds the Sun–Moon **elongation** (the geocentric
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)